src/stories/components/CioPlpGrid/RenderProps.md

```diff
@@ -1,5 +1,18 @@
-| property | type                               | description                        |
-| -------- | ---------------------------------- | ---------------------------------- |
-| data     | `Nullable<ConstructorIOClient>`    | The Constructor IO client instance |
-| status   | `CioClientOptions`                 | The Constructor IO client options  |
-| refetch  | `React.Dispatch<CioClientOptions>` | The Constructor IO options setter  |
+| property   | type                                                                                                                                 | description                             |
```

Comment:
This markdown file is not being used for CioPlpGrid -> Code Examples.mdx

CioPlpGrid -> RenderProps.md has
 - <Markdown>{SearchResultsReturn}</Markdown>

Should it be <Markdown>{RenderProps}</Markdown> instead imported from CioPlpGrid -> RenderProps.md?